addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -221,10 +222,14 @@ var SnippetEditor = Widget.extend({
     /**
      * Notifies all the associated snippet options that the snippet has just
      * been dropped in the page.
+     * 
```

Comment:
```suggestion
     *
```

Reply:
I will change it, thanks for your review